exercises/react-vite/05-routing/01-problem/src/pages/RestaurantList/RestaurantList.tsx

```diff
@@ -0,0 +1,138 @@
+const RestaurantList: React.FC = () => {
+    const restaurants = {
```

Comment:
Someday maybe we can have a pmo API that just returns this stuff so it doesn't have to be copied into the projects as boiler plate.

Reply:
We do have an API and we introduce it later! A lot of the earlier steps have things simplified so later we can introduce concepts like data fetching.